docs/05-technical-reference/00-custom-resources/telemetry-02-logparser.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Telemetry - LogParser
+---
+
+The `logparser.telemetry.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to define a custom log parser in Kyma. To get the up-to-date CRD and show the output in the YAML format, run this command:
+
+```bash
+kubectl get crd logparser.telemetry.kyma-project.io -o yaml
+```
+
+## Sample custom resource
+
+The following LogParser object defines a parser which can parse a log line like: This is an example of parsing a record `{"data":"100 0.5 true This is example"}`.
```

Comment:
Is this the whole log message: `This is an example of parsing a record {"data":"100 0.5 true This is example"}`?

Reply:
Well observed, fixed it